cmd/porter/outputs.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/spf13/cobra"
+)
+
+func buildOutputsCommand(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "outputs",
+		Aliases: []string{"output"},
+		Short:   "Output commands",
+		Long:    "Commands for working with Bundle outputs. TODO",
+	}
+	cmd.Annotations = map[string]string{
+		"group": "resource",
+	}
+
+	cmd.AddCommand(buildOutputListCommand(p))
+	cmd.AddCommand(buildOutputShowCommand(p))
+
+	return cmd
+}
+
+func buildOutputListCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.OutputListOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "list",
+		Short:   "List outputs of a bundle",
+		Long:    "List bundle outputs. TODO",
```

Comment:
Oops, still have a TODO here that needs to be filled out. Maybe

> List outputs generated by a bundle

It doesn't have to be terribly long and it's okay to be the same text as the short.